crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -29,8 +29,6 @@ pub struct Settlement {
     gas: eth::Gas,
     /// The effective gas price of the settlement transaction.
     gas_price: eth::EffectiveGasPrice,
-    /// The address of the solver that submitted the settlement transaction.
-    solver: eth::Address,
```

Comment:
Not strictly related to the task but removed for the sake of completeness so we don't have to rethink how this field should be populated.